### PR TITLE
Update docker image stack

### PR DESCRIPTION
**Pull request by kerkhofsd**

### CODE REVIEW

File: integration-tests/src/test/resources/compose/docker-compose-grafana.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: '2.2'
 
 services:
   grafana:
-    image: hub.xenit.eu/grafana:5.0
+    image: hub.xenit.eu/public/grafana:0.0.1-5
```
Comment (tgeens) [inline integration-tests/src/test/resources/compose/docker-compose-grafana.yml]: @kerkhofsd we need to update & publish our Grafana image first ?

File: integration-tests/alfresco-community-52/overload.gradle [hunk 2]
```diff
@@ -1,4 +1,4 @@
 ext {
     alfrescoBaseWar = 'org.alfresco:alfresco-platform:5.2.g@war'
-    alfrescoBaseImage = 'xeniteu/alfresco-repository-community:5.2.g'
+    alfrescoBaseImage = 'xenit/alfresco-repository-community:5.2.g'
```
Comment (tgeens) [inline integration-tests/alfresco-community-52/overload.gradle]: we should prefix public docker-hub images with `docker.io/`

Comment (kerkhofsd) [reply integration-tests/alfresco-community-52/overload.gradle]: Done

Comment (tgeens) [reply integration-tests/alfresco-community-52/overload.gradle]: I meant `docker.io/xenit/....`

Comment (kerkhofsd) [reply integration-tests/alfresco-community-52/overload.gradle]: Done

**Pull request merged**